native_client/BUILD

```diff
@@ -72,6 +70,7 @@ tf_cc_shared_object(
         "//tensorflow/core/kernels:constant_op",        # Const
         "//tensorflow/core/kernels:immutable_constant_op", # ImmutableConst
         "//tensorflow/core/kernels:identity_op",        # Identity
+        "//tensorflow/core/kernels:softmax_op",         # Softmax
```

Comment:
Shouldn't we also remove `"//tensorflow/core/util/ctc"` above?

Reply:
Good catch.